File: baek_joon/cpp_src/step5/1546.cpp
#include <iostream>
using namespace std;

int main()
{
    int n;
    int max_score;
    int sum = 0;
    int score[10001] = {};

    cin >> n;

    for (int i = 0; i < n; i++)
    {
        cin >> score[i];
        sum += score[i];
    }

    max_score = score[0];

    for (int i = 0; i < n; i++)
    {
        if (score[i] > max_score)
        {
            max_score = score[i];
        }
    }
    
    cout << (double)sum / max_score * 100 / n;
}

File: baekjoon/js/bronzeI/1546.js
// 평균
const fs = require('fs');
const input = fs.readFileSync('/dev/stdin').toString().split('\n');
const num = +input[0];
const score = input[1].split(' ').map(v => +v).sort((a, b) => b - a);
const newScore = score.map(val => val / score[0] * 100);
const sum = newScore.reduce((acc, cur) => acc + cur, 0);

console.log(sum / num);